packages/es-dev-server/src/plugins/browserReloadPlugin.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+import { Plugin } from '../Plugin';
+import debounce from 'debounce';
+
+export interface BrowserReloadPluginConfig {
+  watchDebounce: number;
+}
+
+export function browserReloadPlugin(cfg: BrowserReloadPluginConfig): Plugin {
+  return {
+    serverStart({ fileWatcher, messageChannel }) {
+      if (messageChannel) {
+        function onFileChanged() {
+          console.log('Reloading connected browsers...');
```

Comment:
```suggestion
          console.log('Reloading connected browsers...\n');
```

Reply:
Why the line break? The terminal already creates a line break.

Reply:
instead of the next line

Reply:
Ah ok, make sense.